source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRHostConfig.h

```diff
@@ -0,0 +1,31 @@
+//
+//  ADCIOSAdaptiveHostConfig.h
+//  ADCIOSFramework
+//
+//  Created by jwoo on 6/7/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "TextBlock.h"
+#import "Image.h"
+#import "HostConfig.h"
+
+using namespace AdaptiveCards;
+
+@interface ACRHostConfig:NSObject
+
+- (CGSize) getImageSize:(std::shared_ptr<Image>) img;
+
+- (UIColor* ) getTextBlockColor:(std::shared_ptr<AdaptiveCards::TextBlock>) txtBlock;
+
+- (int) getTextBlockTexSize:(std::shared_ptr<TextBlock>)txtBlock;
```

Comment:
Thanks :-)

---
In reply to: [122551624](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122551624) [](ancestors = 122551624)